app/assets/javascripts/editorToolbar.js

```diff
@@ -69,12 +69,12 @@ $(function() {
           console.log($("#image-upload-progress-container-" + $E.commentFormID));
           $("#image-upload-progress-container-" + $E.commentFormID).show();
           $("#image-upload-text-" + $E.commentFormID).show();
-          $("#dropzone-choose-one-" + $E.commentFormID).hide();
+          $("#choose-one-" + $E.commentFormID).hide();
```

Comment:
removing the dropzone part in the ID slug. choose one is already nested in another, larger dropzone, which I forgot, so this isn't necessary.